Add helm chart

vkobets opened the pull request:
This PR closes #491 #5.
This change adds a helm chart.
Code is prepared to use github.io as a helm registry, https://github.com/helm/chart-releaser-action

It is not tested yet.

I would be happy if someone would test it.

thomaspeitz commented:
Could you please test it yourself first and if you are sure it works, tag me and then I would like to review <3

As I am a private person maintaining this I really depend that I have contributions which take as less private time away from me as possible.

vkobets commented:
> Is the helm chart based on: https://artifacthub.io/packages/helm/mogaal/prometheus-yace-exporter - If not what are differences? :) - Thanks!

your message gave me an idea to merge this repo and then update that code.
So now differences are  
- renamed chart
- updated README.md
- updated templates according to common helm chart template(helm create)
- fixed notes
- fixed healthcheck
- added imagePullSecrets
- added PrometheusRule
